tools/main.cpp

```diff
@@ -70,7 +70,18 @@ int main(int argc, char** argv) {
           antler::update_project,
           antler::validate_project> runner{app};
 
-   CLI11_PARSE(app, argc, argv);
+   try {
+      app.parse(argc, argv);
+   }
+   // This hack fix parsing error handling of an empty command argument. Example: antler-proj build ~/path/proj ""
+   catch (CLI::ExtrasError &e) {
+      if (std::string_view{"The following argument was not expected: "} == e.what()) {
+         return app.exit(CLI::ExtrasError("Empty argument is encountered in the command line", e.get_exit_code()));
+      }
+   }
+   catch (const CLI::ParseError &e) {
```

Comment:
other references in code are similar to this:
```suggestion
   catch (const CLI::ParseError& e) {
```

Reply:
I agree - @mikelik is suggesting the "correct" style.